test/dbus/util-broker.c

```diff
@@ -119,9 +119,39 @@ static int util_append_policy(sd_bus_message *m) {
         return 0;
 }
 
+static int util_method_reload_config(sd_bus_message *message, void *userdata, sd_bus_error *error) {
+        sd_bus *bus = sd_bus_message_get_bus(message);
```

Comment:
Urgs, we usually don't call non-inlined messages as part of initializations. In this case, it doesn't matter, I guess. But often it gets confusing if combined with cleanups.

Reply:
Fair enough, fixed.